wsdl/ver10/schema/common.xsd

```diff
@@ -83,6 +83,23 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 			</xs:element>
 		</xs:sequence>
 	</xs:complexType>
+	<xs:complexType name="FieldOfView">
```

Comment:
I don't think this makes much sense. We have multiple ways to express the coordinate space of a PTZ. I believe the data provided should match the coordinate space configured in the PTZConfiguration

Reply:
There are multiple pan- tilt-coordinate spaces yes, but they all describe where the center of the camera is pointing. To calculate what direction any other pixel of the camera view looks is not possible with the metadata information we have today. 

What we want to be able to do is to calculate which pan- tilt- direction any pixel from the camera view is looking at. To be able to do this calculation from pixel-position to pan-tilt position, we need to know the vertical and horizontal field-of-view (FOV) of the lens. (To make an exact calculation we also need lens distorsion parameters but those are static on a certain type of zoom lens.)

So, as an example, if we want to quickly look more closely at an interesting object in a corner of a zoomed-out camera view, we need to know the current FOV of the camera view to be able to quickly move to this object.

If you have proposals for other FOV-spaces that could be interesting to use, please let us know! 
We use the FOV in degrees on our cameras, so for us it is a de-facto standard. 

(I myself work with PTZ-cameras for Axis Communications. 
It is in that role that I write this change request, even if that may not be visible in my github account.)

Reply:
As a side note: For the use case of centering on an object in view you can use the TranslationSpaceFov, if supported by the device.
See PTZ Service Specification, Annex A.3.2 Pan/tilt Translation Space in FOV

Reply:
Another use-case example is that if this metadata stream is recorded together with the video on a VMS, it is possible to calculate the pan-tilt coordinates of any object in the recorded video from a PTZ camera.

Reply:
> There are multiple pan- tilt-coordinate spaces yes, but they all describe where the center of the camera is pointing. To calculate what direction any other pixel of the camera view looks is not possible with the metadata information we have today.
> 
> What we want to be able to do is to calculate which pan- tilt- direction any pixel from the camera view is looking at. To be able to do this calculation from pixel-position to pan-tilt position, we need to know the vertical and horizontal field-of-view (FOV) of the lens. (To make an exact calculation we also need lens distorsion parameters but those are static on a certain type of zoom lens.)
> 
> So, as an example, if we want to quickly look more closely at an interesting object in a corner of a zoomed-out camera view, we need to know the current FOV of the camera view to be able to quickly move to this object.
> 
> If you have proposals for other FOV-spaces that could be interesting to use, please let us know! We use the FOV in degrees on our cameras, so for us it is a de-facto standard.
> 
> (I myself work with PTZ-cameras for Axis Communications. It is in that role that I write this change request, even if that may not be visible in my github account.)

We also have similar requirements.

Reply:
@svefredrik did you get a chance to reach out to your colleague regarding this.

Reply:
I updated the description a few days ago with a use-case and how this enhancement also will make the ONVIF specification more versatile for those who want to present geolocation in the end.